packages/vscode-boxel-tools/README.md

```diff
@@ -17,11 +10,66 @@ You can also build it [locally](#development).
 
 ## Usage
 
-Open a new window and in the command palette select "Boxel Tools: Attach to Boxel Workspaces".
+### Getting Started
+
+1. Install the extension from the VS Code Marketplace or [build it locally](#development).
+2. Open a new window and in the command palette (F1 or Ctrl+Shift+P), select "Boxel Tools: Attach to Boxel Workspaces".
+3. You will be prompted to log in with your Matrix credentials:
```

Comment:
Maybe we call them Boxel credentials instead of Matrix credentials. We don't use the term Matrix in the app UI AFAIK.